docs/API/functions/Canvas.rst

```diff
@@ -0,0 +1,307 @@
+Canvas
```

Comment:
@aashish24 I have a script in docs/API that builds this out. It checks all of VCS's modules using the `inspect` library, and prints out an RST-formatted link to the appropriate file (https://github.com/embrown/vcs/blob/more_more_documentation/docs/API/build_func_index.py).